python/unitytrainers/curriculum.py

```diff
@@ -1,6 +1,6 @@
 import json
 
-from .exception import UnityEnvironmentException
+from unityagents.exception import UnityEnvironmentException
```

Comment:
I am not sure we want to use UnityEnvironmentException here.

Reply:
Good point. Should I make an exception module under unitytrainers and add a `CurriculumException`?